en/identity-server/next/docs/includes/action-troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+
+```json
+{
+  "logId": "582befe9-6114-4362-8fd4-05496e639fb8",
+  "recordedAt": {
+    "seconds": 1729488306,
+    "nanos": 479103000
+  },
+  "requestId": "d9b5f323-79cb-4a9e-9d84-f83ab7056122",
+  "resultStatus": "SUCCESS",
+  "resultMessage": "Call external service endpoint https://example.com for Pre Issue Access Token action.",
+  "actionId": "process-action-request",
+  "componentId": "action-execution",
+  "configurations": {
+    "action id": "0ab318c4-af38-4190-ae41-75f35ecdf7b6",
+    "action type": "Pre Issue Access Token",
+    "action endpoint": "https://example.com",
+    "action endpoint authentication type": "BASIC"
+  },
+  "logDetailLevel": "APPLICATION"
+}
+```
+
+The following table gives an explanation to each property included in the diagnostic log event.
+
+<table>
+<thead>
+<tr class="header">
+<th>Property</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>logId</td>
+<td>
+<p>Unique ID for each log event</p>
```

Comment:
Check other places whether we have added full-stop at the end of the descriptions in the tables.

Reply:
Updated in https://github.com/wso2/docs-is/pull/4819/commits/a877737fe142a2f9c37e955b8dd2de7749da8f57